recipes/libmatio/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.5.6" %}
+package:
+  name: libmatio
+  version: {{ version }}
+
+source:
+  fn: matio-{{ version }}.tar.gz
+  url: https://github.com/tbeu/matio/releases/download/v{{ version }}/matio-{{ version }}.tar.gz
+  md5: bb53fd06f7ebb74589e9dd684b1b66a6
+
+build:
+  number: 0
+  skip: true     # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+    - gcc
+    - hdf5
+    - zlib
+
+  run:
+    - hdf5
+    - zlib
+
+test:
+  commands:
+    - test -e $PREFIX/include/matio.h  # [unix]
```

Comment:
@jakirkham we should develop a tool to inspect expected libs! Not sure if that should be just a python script here, for conda-forge, of if someone (with better coding skills) could do that in conda-build.

```yaml
test:
   expected-libs:
      - lib1  # and extensions would be automatically appended depending on the OS.
      - lib2  
```

(Just thinking it out loud here...)